pymtl3/passes/backends/verilog/import_/VerilogVerilatorImportConfigs.py

```diff
@@ -85,6 +85,15 @@ class VerilogVerilatorImportConfigs( BasePassConfigs ):
     # Expects a boolean value
     "vl_trace" : False,
 
+    # trace format
+    # Expects a str value
+    # "Vcd" or "Fst
+    "vl_trace_format" : "Vcd",
+
+    # --trace-struct
+    # Expects a boolean value
+    "vl_trace_struct" : False,
```

Comment:
This will expand every struct signal into a "component," which might make finding the target value difficult and make the trace much larger. Tracing all the struct values also has a performance overhead.

Reply:
Got it! That makes sense.